Floofbot/Modules/Fun.cs

```diff
@@ -52,10 +49,33 @@ public async Task AskEightBall([Summary("question")][Remainder] string question)
             };
             Random random = new Random();
             int randomNumber = random.Next(responses.Count);
+            return responses[randomNumber];
+        }
+
+        public static string RespondEightBall(string question)
```

Comment:
This method name is too close to the one above. It might become clearer what to do once EightBall moves into its own class.

Reply:
What about `QuestionEightBall()` ?